src/pages/_app.tsx

```diff
@@ -40,6 +43,20 @@ function CustomApp({
   const router = useRouter()
   const { i18n } = useTranslation()
   const { initialize, trackEvent } = useGTM()
+  const { publicRuntimeConfig } = getConfig()
+  const { socketClientStore, notificationStore } = stores
+
+  useEffect(() => {
+    if (session?.accessToken) {
+      socketClientStore?.initiate(publicRuntimeConfig.API_URL, {
+        transports: ['websocket'],
+        withCredentials: true,
+        query: {
+          token: session?.accessToken || null,
```

Comment:
Don't we want people who aren't logged in to also see the notifications?

Reply:
Not sure about that requirement and also the security problems behind it

Reply:
yeah, I also think the notifications should be visible to everyone

Reply:
what would be a security concern for showing the notification to everyone?